### DISCUSSION TITLE
Sort "Select table to reference" dropdown

### DISCUSSION THREAD

**Discussion by dilly-sixmo** [supabase/supabase#18835] [Feature Requests]
I find it quite annoying that I have to search through this list to find the table I want to reference.  If I have a 100 tables in my system, this becomes a real pain in the butt.  Can this be sorted by name?

![image](https://github.com/supabase/supabase/assets/112413500/bb9f0eef-3712-4cb6-a52e-4abdd3649aa4)

**Comment by Hallidayo** [comment]
Hi @dilly-sixmo - I've open a pull request for this: #18926

**Comment by bombillazo** [comment]
Thanks for creating this, Id say the same for when choosing a column data type. Super annoying to be scrolling around looking for the type.

**Reply by Hallidayo** [reply]
Hi @bombillazo - I'll look to get a fix out for this.
